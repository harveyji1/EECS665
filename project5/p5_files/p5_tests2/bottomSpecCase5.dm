p : bool;
f : (x : int) bool { return true; }

main : () void {
    d : int;
    d = 1 + p();                 // p() is an error; the + is OK
}