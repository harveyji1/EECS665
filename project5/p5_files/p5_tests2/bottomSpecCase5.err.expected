FATAL [6,13]-[6,14]: Attempt to call a non-function
Type Analysis Failed
